src/main/java/mate/academy/rickandmorty/service/character/CharacterService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.rickandmorty.service.character;
+
+import java.util.List;
+import mate.academy.rickandmorty.dto.character.CharacterDto;
+
+public interface CharacterService {
+    CharacterDto randomCharacter();
```

Comment:
```suggestion
    CharacterDto getRandomCharacter();
```
method names should be verbs/verb phrases